templates/jdbc-postgresql-repository.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type" : "db",
+            "connectionUrl": "jdbc:postgresql://${fcrepo.postgresql.host:localhost}:${fcrepo.postgresql.port:5432}/fcrepo",
```

Comment:
Same changes as in the MySQL, or you could combine the two into a single template file and change the `jdbc:mysql` to `jdbc:postgresql` depending on the database type. Your call.

Reply:
I'm going to keep the logic simple (right now we just choose the file based on the file-name prefixing the repository.json with which db we are using). Using one template for both the mysql and postgres would require two tasks with when statements.